src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,175 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.User;
+import core.basesyntax.validators.RegistrationValidatorException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final int AGE_LESS_MIN = 17;
+    private static final int AGE_LESS_ZERO = -10;
+    private RegistrationServiceImpl registrationService;
+    private StorageDaoImpl storageDao;
+    private User testUser;
+
+    @BeforeEach
+    void init() {
+        registrationService = new RegistrationServiceImpl();
+        storageDao = new StorageDaoImpl();
+        testUser = new User();
+    }
+
+    @Test
+    void register_nullUser_notOk() {
+        assertThrows(RegistrationValidatorException.class, () -> {
+            registrationService.register(null);
+        });
+    }
+
+    @Test
+    void register_nullLogin_notOk() {
+        testUser.setLogin(null);
```

Comment:
testUser has all fields null - look at init() method
set all correct fiels in init() and than set one incorrect in test methods.